Library/Homebrew/formula_auditor.rb

```diff
@@ -582,15 +582,15 @@ def audit_eol
       metadata = SharedAudits.eol_data(name, formula.version.major)
       metadata ||= SharedAudits.eol_data(name, formula.version.major_minor)
 
-      return if metadata.blank? || metadata["eol"] == false
+      return if metadata.blank? || metadata["eol"].blank?
 
       see_url = "see #{Formatter.url("https://endoflife.date/#{name}")}"
-      if metadata["eol"] == true
+      if metadata["eol"].present?
```

Comment:
This seems to always be true given that we check for `blank?` above?

Reply:
D'oh.